client/src/theme.scss

```diff
@@ -122,12 +122,12 @@ $mt-theme: mat-light-theme($mt-palette-primary, $mt-palette-accent);
   border-radius: 5px;
 }
 
-@media only screen and (max-width: 600px) {
-  ::ng-deep .cdk-overlay-pane {
-    max-width: none !important;
-    height: 100%;
-  }
-  ::ng-deep .mat-dialog-container {
-    padding: 0px;
-  }
-}
+// @media only screen and (max-width: 600px) {
+//   ::ng-deep .cdk-overlay-pane {
+//     max-width: none !important;
+//     height: 100%;
+//   }
+//   ::ng-deep .mat-dialog-container {
+//     padding: 0px;
+//   }
+// }
```

Comment:
We need this for other popups that appear in the app such as edit user. Note, this only affects the mobile version which should have the same behaviour.

Reply:
It actually not good for mobile and certainly not good to use ng-deep. It would be better to change the dialog in the edit-user component and not for all components.